docs/telemetry.md

```diff
@@ -0,0 +1,76 @@
+| promscale_metrics_total                                     | 0            |
+| promscale_os                                                | linux        |
+| promscale_os_id                                             | ubuntu       |
+| promscale_os_machine                                        | x86_64       |
+| promscale_os_node_name                                      | ubuntu-focal |
+| promscale_os_sys_name                                       | Linux        |
+| promscale_os_version                                        | 20.04        |
+| promscale_packager                                          | unknown      |
+| promscale_promql_query_execution_time_p50                   | 0.0000       |
+| promscale_promql_query_execution_time_p90                   | 0.0000       |
+| promscale_promql_query_execution_time_p95                   | 0.0000       |
+| promscale_promql_query_execution_time_p99                   | 0.0000       |
+| promscale_promql_query_remote_read_batch_execution_time_p50 | 0.0000       |
+| promscale_promql_query_remote_read_batch_execution_time_p90 | 0.0000       |
+| promscale_promql_query_remote_read_batch_execution_time_p95 | 0.0000       |
+| promscale_promql_query_remote_read_batch_execution_time_p99 | 0.0000       |
+| promscale_promql_telemetry_evaluation_duration_seconds      | 1.027        |
+| promscale_trace_dependency_requests_executed_total          | 0            |
+| promscale_trace_query_execution_time_p50                    | 0.0000       |
+| promscale_trace_query_execution_time_p90                    | 0.0000       |
+| promscale_trace_query_execution_time_p95                    | 0.0000       |
+| promscale_trace_query_execution_time_p99                    | 0.0000       |
+| promscale_trace_query_requests_executed_total               | 0            |
+| promscale_traces_spans_bytes_total                          | 65536        |
+| promscale_traces_spans_total_approx                         | 0            |
+| promscale_traces_total_approx                               | 0            |
+| promscale_version                                           | 0.8.0        |
+
+
+This data together with usage data collected by
+[TimescaleDB](https://docs.timescale.com/timescaledb/latest/how-to-guides/configuration/telemetry/) 
+is sent to our servers every 24 hours.
+
+## Disable Telemetry
+
+Promscale usage telemetry is collected through TimescaleDB's telemetry feature.
+
+You can [exclude specific items](https://docs.timescale.com/timescaledb/latest/how-to-guides/configuration/telemetry/#change-what-is-included-the-telemetry-report) you are not confortable sharing
+ or [completely disable the feature](https://docs.timescale.com/timescaledb/latest/how-to-guides/configuration/telemetry/#disable-telemetry).
+
+Note that data in the table above will continue to be refreshed (but not shared) 
+even after disabling the telemetry feature. Promscale usage telemetry is refreshed every hour.
```

Comment:
If telemetry is disabled, we do not refresh it. We totally do not create the telemetry engine.

Relevent code: https://github.com/timescale/promscale/blob/374c8435899efbe2ecb3d1687c7b47cd2785253c/pkg/telemetry/telemetry.go#L54

Reply:
Thanks @Harkishen-Singh. I've removed the note about telemetry still being collected.